devtools/gammacli/json_client_test.go

```diff
@@ -0,0 +1,242 @@
+package gammacli
+
+import (
+	"encoding/hex"
+	"github.com/orbs-network/orbs-network-go/crypto/digest"
+	"github.com/orbs-network/orbs-network-go/crypto/signature"
+	"github.com/orbs-network/orbs-network-go/test/builders"
+	"github.com/orbs-network/orbs-network-go/test/crypto/keys"
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
+	"github.com/orbs-network/orbs-spec/types/go/protocol/client"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestConvertJSONTransactionWithArgumentsOfTypeString(t *testing.T) {
```

Comment:
There's a lot of repetition in these tests. Either extract the assertions to matcher functions, or only test the different parts in the second test and onwards. 

The best solution, though, would be to split the method argument conversion logic out of `ConvertJSONTransactionToMemBuff` and test it separately.

Reply:
Refactored accordignly